LaunchAtLogin/copy-helper.sh

```diff
@@ -9,7 +9,7 @@ mkdir -p "$helper_dir"
 cp -rf "$origin_helper_path" "$helper_dir/"
 
 defaults write "$helper_path/Contents/Info" CFBundleIdentifier -string "$PRODUCT_BUNDLE_IDENTIFIER-LaunchAtLoginHelper"
-codesign --force --entitlements="$CODE_SIGN_ENTITLEMENTS" --sign="$EXPANDED_CODE_SIGN_IDENTITY_NAME" "$helper_path"
+codesign --force --entitlements="$CODE_SIGN_ENTITLEMENTS" -o runtime --sign="$EXPANDED_CODE_SIGN_IDENTITY_NAME" "$helper_path"
```

Comment:
Use the long-flag `--options`, for readability, and `=` between flag and value.

Reply:
Alright, done.